<?xml version="1.0" encoding="utf-8" standalone="yes" ?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>Pyenv on 程序猿的小本本</title>
    <link>http://www.gsymy.com/tags/pyenv/</link>
    <description>Recent content in Pyenv on 程序猿的小本本</description>
    <generator>Hugo -- gohugo.io</generator>
    <language>en-us</language>
    <managingEditor>chennqqi@qq.com (chennqqi)</managingEditor>
    <webMaster>chennqqi@qq.com (chennqqi)</webMaster>
    <copyright>(c) 2016 gsymy.com.</copyright>
    <lastBuildDate>Thu, 25 Aug 2016 13:35:57 +0800</lastBuildDate>
    
	<atom:link href="http://www.gsymy.com/tags/pyenv/index.xml" rel="self" type="application/rss+xml" />
    
    
    <item>
      <title>pyenv rehash problem</title>
      <link>http://www.gsymy.com/2016/08/25/pyenv_rehash_problem.html</link>
      <pubDate>Thu, 25 Aug 2016 13:35:57 +0800</pubDate>
      <author>chennqqi@qq.com (chennqqi)</author>
      <guid>http://www.gsymy.com/2016/08/25/pyenv_rehash_problem.html</guid>
      <description>#pyenv rehash problem
description:
pyenv: cannot rehash: /home/XXX/.pyenv/shims/.pyenv-shim exists  fixup:
rm /home/XXX/.pyenv/shims/.pyenv-shim exists pyenv rehash &amp;amp;&amp;amp; hash -r pyenv rehash &amp;amp;&amp;amp; rehash  note:
replace XXX with your real username  reference:
http://stackoverflow.com/questions/29753592/pyenv-shim-not-created-when-installing-package-using-setup-py</description>
    </item>
    
  </channel>
</rss>